src/renderer/services/neo.js

```diff
@@ -655,16 +655,24 @@ export default {
         store.commit('setSendInProgress', true);
         sendPromise
           .then((res) => {
-            if (!res || !res.tx) {
+            if (!res || !res.tx || !res.response) {
+              this.$store.commit('setSendInProgress', false);
               return reject('Failed to create transaction.');
             }
-            alerts.success(`Transaction Hash: ${res.tx.hash} Sent, waiting for confirmation.`);
+
+            if (res.response.result !== true) {
+              this.$store.commit('setSendInProgress', false);
+              return reject('Transaction rejected by NEO network.');
+            }
+
+            alerts.success(`Transaction Hash: ${res.tx.hash} Successfully Sent, waiting for confirmation.`);
```

Comment:
Eventually we need to move all of these alerts out of these services. They are moving to a shared npm package to be used with the mobile/web app.